examples/pyfunc/train_custom_model.py

```diff
@@ -0,0 +1,64 @@
+import os
```

Comment:
@sniafas Can you add this file in `mlflow/tests/examples/test_examples.py`?

https://github.com/mlflow/mlflow/blob/8c796acd04859b885643890e0fcf5be19487713e/tests/examples/test_examples.py#L195